src/api/java/com/ldtteam/structurize/api/util/ItemStackUtils.java

```diff
@@ -85,5 +93,46 @@ public static int getSize(final ItemStack stack)
 
         return stack.getCount();
     }
+
+    /**
+     * Get the list of required resources for entities.
+     *
+     * @param entity the entity object.
+     * @param pos the placer pos..
+     * @return a list of stacks.
+     */
+    public static List<ItemStack> getListOfStackForEntity(final Entity entity, final BlockPos pos)
+    {
+        if (entity != null)
+        {
+            final List<ItemStack> request = new ArrayList<>();
+            if (entity instanceof ItemFrameEntity)
+            {
+                final ItemStack stack = ((ItemFrameEntity) entity).getDisplayedItem();
+                if (!ItemStackUtils.isEmpty(stack))
+                {
+                    stack.setCount(1);
+                    request.add(stack);
+                }
+                request.add(new ItemStack(Items.ITEM_FRAME, 1));
+            }
+            else if (entity instanceof ArmorStandEntity)
```

Comment:
Would a check for IItemHandler be a good idea here?

Reply:
Good question, I think if we handle itemHandler checks here that would require quite a bit of testing. This used to be a bit more generic and caused a lot of issues with other entities, that's why I removed it and made it more explicit. For the time being that is the best way to handle it.